native/cocos/platform/mac/AppDelegate.mm

```diff
@@ -44,6 +44,7 @@ - (void)createLeftBottomWindow:(NSString*)title width:(int)w height:(int)h {
 }
 
 - (void)createWindow:(NSString*)title xPos:(int)x yPos:(int)y width:(int)w height:(int)h {
+#if !defined(CC_SERVER_MODE)
```

Comment:
Can add macro to control window or not, not in the implementation? For example:
```c++
#if !defined(CC_SERVER_MODE)
[xxx createWidow];
#endif 
```

Reply:
ok